src/deepsparse/v2/text_generation/prep_for_prefill.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Any, Optional
+
+from deepsparse.v2.operators import Operator
+from deepsparse.v2.utils import Context, InferenceState, PipelineState
+
+
+__all__ = ["PrepareforPrefill"]
+
+
+class PrepareforPrefill(Operator):
```

Comment:
will we need these "prep" operators after the refactor?

Reply:
so far they're responsible for doing things needed before prompt_inference (create a kv_cache) and before generation (set-up the token_generator for example) so they should still be here.